src/pages/tBTC/Bridge/components/DepositDetailsStep.tsx

```diff
@@ -53,108 +33,66 @@ type CommonStepProps = Pick<BridgeProcessStepProps, "onComplete"> & {
 }
 
 export const Step1: FC<
```

Comment:
Non-blocking:

While we're at it can we also add `DepositDetails` prefix to step components? Like `DepositDetailsStep1`, `DepositDetailsStep2` etc.?

Reply:
Ref commit: be7d18d34a6580c841ff9af8f54e5d693b3402f8